creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -21,12 +41,43 @@ const MetricTypes = Object.freeze({
 let MetricNames = {
   SYNC_QUEUE_JOBS_TOTAL_GAUGE: 'sync_queue_jobs_total',
   ROUTE_POST_TRACKS_DURATION_SECONDS_HISTOGRAM:
-    'route_post_tracks_duration_seconds'
+    'route_post_tracks_duration_seconds',
+  ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM:
+    'issue_sync_seconds_waiting_for_completion_histogram'
 }
 MetricNames = Object.freeze(
   _.mapValues(MetricNames, (metricName) => NamespacePrefix + metricName)
 )
 
+// Uses an implementation similar to the Golang client's ExponentialBucketsRange function:
```

Comment:
added. I didn't actually see that very old issue btw so my reasoning was just that the other implementation works for everyone using the official golang client and isn't too hard to port over to JS because it's mostly just math

Reply:
no i totally support your decision, just meant worth commenting why